eng/pipelines/templates/stages/archetype-java-release-batch.yml

```diff
@@ -503,3 +503,5 @@ stages:
                   DocRepoLocation: $(DocRepoLocation)
                   TargetDocRepoName: ${{parameters.TargetDocRepoName}}
                   TargetDocRepoOwner: "azure-sdk"
+
```

Comment:
Don't you also need to add this to https://github.com/Azure/azure-sdk-for-java/blob/main/eng/pipelines/templates/stages/archetype-java-release-patch.yml#L251

Reply:
Right now I only want to fail the nightly pipeline and `archetype-java-release-patch` doesn't have a nightly integration stage.